src/api/index.ts

```diff
@@ -6,6 +6,14 @@ const readApi = async (path: string): Promise<a.Root> => {
   return api;
 };
 
+const getMimeType = (content: a.Content): a.MimeType | undefined => {
+  if (content["application/json"]) {
+    return a.newMimeType({
+      items: Object.keys(content["application/json"].schema).length,
+    });
+  }
+};
```

Comment:
Note that in the special-case of JSON, `application/json` and `text/json` are synonymous (since JSON prescribes UTF8 and ignores newlines). We can thus support both.

Reply:
Added support for text/json as well. This meant i could remove a lot of if(var === "application/json")

https://github.com/XenitAB/apidelta/pull/14/commits/f4fff90e8d058e32262067d1e4b9070cc2d7b3ec